src/ex6/server/client/src/App.js

```diff
@@ -0,0 +1,12 @@
+import "./App.css"
+import { Home } from "./pages/Home/Home"
```

Comment:
the folder structure for your project is a bit confusing
![image](https://user-images.githubusercontent.com/36815567/178156412-10eaaf36-6634-4902-9072-b37c8693d1fe.png)

Reply:
the root directory is called server, where inside there's another server folder and the client